### PR TITLE
Add Arcade-based build

### PR DESCRIPTION
**Pull request by directhex**
This should approximately mirror the Google binaries consumed by our existing emsdk

### CODE REVIEW

**Comment by directhex** [comment]
The Mac builds are SO SLOW, ARM64 moreso.

File: NuGet.config [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <solution>
+    <add key="disableSourceControlIntegration" value="true" />
+  </solution>
+  <packageSources>
+    <clear />
+    <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
+    <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
+    <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
+    <add key="dotnet8" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" />
+    <add key="dotnet8-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8-transport/nuget/v3/index.json" />
```
Comment (akoeplinger) [inline NuGet.config:12]: I think we don't actually need the dotnet8 feeds, one less place to update when doing version bumps

**Review by akoeplinger — APPROVED** [review]

File: NuGet.config (hunk 1 above)
Comment (directhex) [reply NuGet.config:12]: We need the transport feed, for Python. At that point we need to update the version in that feed anyway. And I think some of the SDK stuff like Sdk.Traversal is in a nupkg in the main dotnet8 feed.

Comment (akoeplinger) [reply NuGet.config:12]: Ah, ok.

Comment (akoeplinger) [reply NuGet.config:12]: The traversal stuff is in dotnet-public so having dotnet8-transport should be enough

**Pull request merged**